dirs/dirs.go

```diff
@@ -240,7 +240,7 @@ func SetRootDir(rootdir string) {
 	GlobalRootDir = rootdir
 
 	isInsideBase, _ := isInsideBaseSnap()
-	if !isInsideBase && release.DistroLike("fedora", "arch", "archlinux", "manjaro", "antergos") {
+	if !isInsideBase && release.DistroLike("fedora", "arch", "archlinux", "manjaro", "manjaro-arm", "antergos") {
```

Comment:
Maybe a XXX here that we should look at manjaro-arm again if it grew DISTRO_LIKE= now?

Reply:
Based on my discussion I don't think upstream is strongly considering calling it ID_LIKE=manjaro for now.